src/common/config.h

```diff
@@ -13,7 +13,7 @@
 //--------------------------------------
 //marlin api config
 enum {
-    MARLIN_MAX_CLIENTS = 3,    // maximum number of clients registered in same time
+    MARLIN_MAX_CLIENTS = 5,    // maximum number of clients registered in same time
```

Comment:
Why does this PR increase this by two, but adds one client only?

Reply:
I'm not entirely sure (I'm not the author). I vaguely remember asking @marekmosna-prusa3d the same thing at the original review, some months ago and he said something he wanted one extra for some reason.

I can clean it up and remove the one extra. Should I?

Reply:
In case we don't think it is necessary, then yes, please remove the extra one.

Reply:
I've grepped the code and I'm quite confident there are just 4 being created (two wui, one in guimain and one in thread_measurement).